doc/source/structures/misc/resource_transfer.rst

```diff
@@ -0,0 +1,100 @@
+.. _resource_transfer:
+
+Resource Transfer 
+=================
+
+Usage is covered elsewhere <../commands/resource_transfer.html>`__
```

Comment:
I am bad, i never installed it so i have been just guessing, i need to just pull the trigger on it :(